src/calendar/calendar.tsx

```diff
@@ -278,13 +294,14 @@ export default class Calendar extends Component<Props, State> {
       // @ts-ignore: No overload matches this call
       <Month
         customRender={renderMonth}
-        renderDay={renderDay}
+        renderWeekDay={renderWeekDay}
         renderWeek={renderWeek}
         renderDaysOfWeek={renderDaysOfWeek}
         renderDayOfWeek={renderDayOfWeek}
         activeMonth={this._activeMonth()}
         blockClassName={blockClassName}
         daysOfWeek={daysOfWeek}
+        getWeekDayFormatted={getWeekDayFormatted}
```

Comment:
maybe those `getWeekDayFormatted` and `getISODate` can be parts of the `WeekDay` implementation? and if you write your own `renderWeekDay` then you can reuse them from `utils`? (not critical or something, just thoughts, so please ignore them if they are not relevant)